packages/core/admin/admin/src/content-manager/components/PreviewWysiwyg/Wrapper.js

```diff
@@ -4,7 +4,7 @@ const Wrapper = styled.div`
   position: absolute;
   top: 0;
   width: 100%;
-  height: 100%;
+  height: calc(100% - 3rem);
```

Comment:
This resolves the issues with the bottom content of the expanded preview panel being cut off based on the height of the header.

<img width="1207" alt="Screenshot 2023-01-25 at 8 56 54 AM" src="https://user-images.githubusercontent.com/1501599/214509688-913ac9e6-fdd6-4306-b868-05298bd9934b.png">